playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -585,6 +585,16 @@ EDXAPP_PRIVATE_REQUIREMENTS:
     # Caliper and xAPI event routing plugin
     - name: edx-event-routing-backends==5.5.6
 
+# List of additional npm packages that should be installed into the
+# edxapp virtual environment.
+# `name` (required), `version` (optional), and `extra_args` (optional)
+# Example:
+# EDXAPP_EXTRA_NPM_REQUIREMENTS:
+#   - name: mypackage
+#     version: ^1.0.0
+#   - name: git+https://git.myproject.org/MyProject#egg=MyProject
+EDXAPP_EXTRA_NPM_REQUIREMENTS: []
```

Comment:
- Separately, I'm going to be deprecating and removing `EDXAPP_EXTRA_REQUIREMENTS` in favor of `EDXAPP_PRIVATE_REQUIREMENTS`.
- Can we rename to `EDXAPP_PRIVATE_NPM_REQUIREMENTS` for consistency?

Reply:
All set! 🫡